### ISSUE TITLE
First tab into app doesn't show focus indicator [accessibility]

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2020]
Other accessibility-related changes to the DOM order might affect this, but recording it now to keep track. 

On first tab into the application, the first element in focus (sidebar toggle in first window) does not get a visible focus indicator. Second and sequential tabs get the focus highlight, and returning to the original element gets the highlight. Only the first entrance after an app reload does not.

Gif shows a black outline around the button that is actually in focus, expectation is that the grey highlight should also appear. 

![no focus on first tab](https://user-images.githubusercontent.com/6945691/53665779-38709380-3c21-11e9-9096-bc3bb83ad82e.gif)

**Comment by mejackreed** [comment]
Resolved with other keyboard focus work.
